.github/workflows/e2e-browser.yml

```diff
@@ -29,12 +29,16 @@ jobs:
           node-version: 16.x
           cache: npm
       - run: npm ci
+        if: github.actor != 'dependabot[bot]'
       - run: npm run bootstrap -- --ci
```

Comment:
do we actually need this still in CI?

Reply:
I don't think so, since this is in the `package.json`: "postinstall": "lerna bootstrap --ci && npm run build". I'll change this in a separate PR.